bin/gather_pipelines

```diff
@@ -73,11 +83,12 @@ APIRESOURCES=(eventlistener taskrun pipelinerun)
 for APIRESOURCE in ${APIRESOURCES[@]}; do
     NAMESPACES=$(oc get ${APIRESOURCE} --all-namespaces=true --ignore-not-found -o jsonpath='{range .items[*]}{@.metadata.namespace}{"\n"}{end}' | uniq)
     for NAMESPACE in ${NAMESPACES[@]}; do
-        mkdir -p ${LOGS_DIR}/namespaces/${NAMESPACE}/${APIRESOURCE}/logs
         RESOURCES=$(oc get ${APIRESOURCE} --namespace=${NAMESPACE} --ignore-not-found -o jsonpath='{range .items[*]}{@.metadata.name}{"\n"}{end}')
         for RESOURCE in ${RESOURCES[@]}; do
             echo "gather_pipelines:$LINENO] collecting ${APIRESOURCE} logs in ${NAMESPACE} .."
-            tkn ${APIRESOURCE} logs ${RESOURCE} -n ${NAMESPACE} >${LOGS_DIR}/namespaces/${NAMESPACE}/${APIRESOURCE}/logs/${RESOURCE}.log
+            mkdir -p ${LOGS_DIR}/namespaces/${NAMESPACE}/${APIRESOURCE}/${RESOURCE}
+            tkn ${APIRESOURCE} logs ${RESOURCE} -n ${NAMESPACE} >${LOGS_DIR}/namespaces/${NAMESPACE}/${APIRESOURCE}/${RESOURCE}/${RESOURCE}.log
+            oc get ${APIRESOURCE} ${RESOURCE} -n ${NAMESPACE} -oyaml > ${LOGS_DIR}/namespaces/${NAMESPACE}/${APIRESOURCE}/${RESOURCE}/${RESOURCE}.yaml
```

Comment:
yaml file for namespaced resources are already collected [here](https://github.com/openshift-pipelines/must-gather/blob/6b6f153b98b820307f07ae38936c74a30bd13f14/bin/gather_pipelines#L22)

Reply:
I didn't get you 🤔

Reply:
Yeah they have all tekton resources under one yaml i.e.

![image](https://github.com/openshift-pipelines/must-gather/assets/32545638/2283960a-e523-4049-a81f-6a3af503ac16)


and for this I'm creating separate directory for each `pipelinerun, pods, taskruns` with their yamls and logs i.e. 

![image](https://github.com/openshift-pipelines/must-gather/assets/32545638/fac3788d-73b8-439b-aae1-ecf058e987b2)

Will this break omc ?

Reply:
I don't remember to be honest, but you can [test by yourself](https://github.com/gmeghnag/omc/releases/latest) :)